mgtool/build.go

```diff
@@ -12,21 +12,36 @@ import (
 )
 
 func GoInstall(ctx context.Context, goPkg, version string) (string, error) {
-	toolDir, err := filepath.Abs(mgpath.Tools())
-	if err != nil {
-		return "", err
-	}
-	executable := filepath.Join(toolDir, goPkg, version, "bin", filepath.Base(goPkg))
+	executable := filepath.Join(mgpath.Tools(), "go", goPkg, version, "bin", filepath.Base(goPkg))
+	symlink := filepath.Join(mgpath.Tools(), mgpath.BinDir, filepath.Base(goPkg))
 
 	// Check if executable already exist
 	if _, err := os.Stat(executable); err == nil {
-		return executable, nil
+		if err := createSymlink(executable, symlink); err != nil {
+			return "", err
+		}
+		return symlink, nil
 	}
 	goPkgVer := fmt.Sprintf("%s@%s", goPkg, version)
 	os.Setenv("GOBIN", filepath.Dir(executable))
 	logr.FromContextOrDiscard(ctx).Info(fmt.Sprintf("Building %s...", goPkgVer))
 	if err := sh.RunV("go", "install", goPkgVer); err != nil {
 		return "", err
 	}
-	return executable, nil
+	if err := createSymlink(executable, symlink); err != nil {
+		return "", err
```

Comment:
maybe wrap this err to make it easier to troubleshoot:

```suggestion
		return "", fmt.Errorf("failed to create symlink: %w", err)
```

Reply:
IMO the `createSymlink` function should talk about failure to add symlink

Reply:
The error I got, from when trying to create a symlink when one already existed was clear enough. I think we can stick with what we got